src/tests/stub_ipc_Forwarder.cc

```diff
@@ -10,8 +10,9 @@
 #include "ipc/Forwarder.h"
 
 //Avoid linker errors about Ipc::Forwarder
+void foo_stub_ipc_forwarder();
 void foo_stub_ipc_forwarder()
 {
-    Ipc::Forwarder foo(NULL,1.0);
-}
+    Ipc::Forwarder foo(nullptr,1.0);
+};
```

Comment:
This is an out of scope change. I recommend undoing, but if you insist on keeping it, then please remove the extra trailing `;`.

Reply:
reverting